Orange/widgets/evaluate/owparameterfitter.py

```diff
@@ -145,9 +146,9 @@ class FitterPlot(PlotWidget):
 
     def __init__(self):
         super().__init__(enableMenu=False)
-        self.__bar_item_tr: pg.BarGraphItem = None
-        self.__bar_item_cv: pg.BarGraphItem = None
-        self.__data: list[ScoreType] = None
+        self.__bar_item_tr: Optional[pg.BarGraphItem] = None
```

Comment:
I swear I'm not nitpicking, it's just because PyCharm marks this `None` as invalid. :)